resource/menu/client/cl_base.lua

```diff
@@ -194,10 +196,10 @@ RegisterNetEvent('txcl:heal', function()
   local pos = GetEntityCoords(ped)
   local heading = GetEntityHeading(ped)
   if IsEntityDead(ped) then
-    NetworkResurrectLocalPlayer(pos[1], pos[2], pos[3], heading, false, false)
+       NetworkResurrectLocalPlayer(pos[1], pos[2], pos[3], heading, 0, false,false,false)
```

Comment:
Same thing. NetworkResurrectLocalPlayer does not accept that many parameters.

Reply:
txAdmin is not just for FiveM hence why the extra parameters
https://rdr3natives.github.io/?_0xEA23C49EAA83ACFB